### DISCUSSION TITLE
How can I do to close Quartz log  info about NLog

### DISCUSSION THREAD

**Discussion by fancky2019** [quartznet/quartznet#949] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by wtujvk** [comment]
// to close Quartz log 
Quartz.Logging.LogProvider.IsDisabled = true;

**Reply by fancky2019** [reply]
> Quartz.Logging.LogProvider.IsDisabled = true;

**Comment by lahma** [comment]
Another option is do configure per logging implementation the level of logging so that Quartz won't show up. I would suggest looking at least the warning level to see if there are real problems around.

**Reply by fancky2019** [reply]
> Another option is do configure per logging implementation the level of logging so that Quartz won't show up. I would suggest looking at least the warning level to see if there are real problems around.
